source/_posts/2023-04-18-home-assistant-os-release-10.markdown

```diff
@@ -0,0 +1,110 @@
+date_formatted: "April 18, 2023"
+author: Stefan Agner
+comments: true
+categories:
+  - Announcements
+og_image: /images/blog/2022-05-16-os8/social.png
+---
+
+<a href='https://github.com/home-assistant/operating-system/releases/tag/10.0'>
+<img title='Home Assistant OS Release 10' alt='Home Assistant OS Release 10 Logo'
+     src='/images/blog/2022-05-16-os8/social.png' style='border: 0;box-shadow: none;'>
+</a>
+<br><br>
+
+**Highlights**:
+
+- Support for Hardkernel ODROID-M1
+- Improved datadisk feature:
+  - Improved reliability
+  - Move from one datadisk to a new datadisk is now supported
+- Improved memory management to improve overall device performance, especially under low memory situations
+- Updated software packages: Linux, Docker, BlueZ, NetworkManager
+
+For existing installations, no manual intervention is needed! You can safely
+update without reading these rather technical release notes.
+
+## Table of contents
+
+- [Table of contents](#table-of-contents)
+- [New board support: Hardkernel ODROID-M1](#new-board-support-hardkernel-odroid-m1)
+- [Improved datadisk feature](#improved-datadisk-feature)
+- [Advanced memory management](#advanced-memory-management)
+- [Updated software packages](#updated-software-packages)
+
+## New board support: Hardkernel ODROID-M1
+
+Home Assistant OS 10 supports the Hardkernel ODROID-M1 single board computer!
+Most notably the ODROID-M1 supports NVMe SSD storage natively, which makes it
+another great choice to run Home Assistant on. With its quad-core CPU with up
+to 2 GHz and up to 8 GB of memory it is suitable even for demanding Home
+Assistant installations.
+
+Currently, Home Assistant can be booted off an SD-card or an eMMC. Note however
+that a new boot firmware (Petitboot) is required to boot from eMMC (see
+the board-specific documentation). NVMe SSD boot is currently not supported
+by the boot firmware. However, an NVMe SSD can be used through the datadisk
+feature.
+
+<img src='/images/blog/2023-04-18-os10/hardkernel-odroid-m1.jpg' alt='Hardkernel ODROID-M1 single board computer with NVMe SSD plugged in'>
+
+## Improved datadisk feature
+
+The datadisk feature allows extending storage by adding an external disk.
+When using the datadisk feature, all commonly read and written data is moved
+to that storage, just Home Assistant OS itself remains on the existing storage
+(e.g. SD-card or eMMC). Home Assistant OS is a read-only operating system - the
+OS partitions are only written to when updating the operating system. This makes
+sure that there is minimal wear on the existing storage.
+
+In this OS release, together with the latest version of the Supervisor, the
+datadisk feature got more user-friendly: The datadisk choice lists the actual
+model of available data disks. In situations where previously an attached disk
+would not show up, the disk will now be reliably detected and available as the
+new datadisk. Now it is also possible to move from one data disk to another:
```

Comment:
```suggestion
new data disk. Now it is also possible to move from one data disk to another:
```

Reply:
In the UI we use datadisk as one word. Shouldn't we stay consistent?

Reply:
As discussed elsewhere, compound nouns should be written with a space.